web/views.py

```
@@ -30,6 +30,9 @@
 from django.views.decorators.csrf import csrf_exempt
 from django.views.decorators.http import require_GET, require_POST
 from django.views.generic import CreateView, DeleteView, ListView, UpdateView
+from django.utils.timezone import now
+from django.utils.timezone import make_aware, now
```

Ruff (0.8.2)

Redefinition of unused `now` from line 33

Remove definition: `now`

(F811)